spec/index.html

```diff
@@ -804,12 +804,19 @@ <h2>Literals</h2>
 
     <p><dfn data-local-lt="term-equal">Literal term equality</dfn>:
       Two literals are term-equal (the same <a>RDF literal</a>)
-      if and only if the two <a>lexical forms</a>,
-      the two <a>datatype IRIs</a>,
-      the two <a>language tags</a> (if any), and
-      the two <a>base directions</a> (if any) compare equal,
-      using <a data-cite="I18N-GLOSSARY#dfn-case-sensitive">case sensitive matching</a>
-      (see description of string comparison in <a href="#rdf-strings" class="sectionRef"></a>).
+      if and only if:
+      <ol>
+        <li>the two <a>lexical forms</a> compare equal</li>
+        <li>the two <a>datatype IRIs</a> compare equal</li>
+        <li>the two <a>language tags</a> (if any) compare equal</li>
+        <li>the two <a>base directions</a> (if any) compare equal</li>
+      </ol>
```

Comment:
There's no reason the various comparisons need to be done in the order of this presentation, so an unordered list is more appropriate.
```suggestion
      <ul>
        <li>the two <a>lexical forms</a> compare equal</li>
        <li>the two <a>datatype IRIs</a> compare equal</li>
        <li>the two <a>language tags</a> (if any) compare equal</li>
        <li>the two <a>base directions</a> (if any) compare equal</li>
      </ul>
```